developer/language/guide/multi-platform.php

```diff
@@ -24,28 +27,27 @@
     Creating a touch keyboard layout for Amharic</a> guide for an in-depth discussion of the benefits of touch layouts
 and some philosophy on touch keyboard design.</p>
 
+<p>While the presented layout will differ according to the physical device it is running on, the keyboard logic implemented in
+  keyboard rules will be available to all platforms. This includes functionality such as reordering of input, normalization,
+  and prevention of invalid sequences.</p>
+
 <p>In the case that a keyboard should have special features or handling on a given platform, consider use of the
   <a href="/developer/language/reference/platform"><code class="language-keyman">platform()</code></a> statement to allow your keyboard
 the flexibility necessary to accomplish those goals.</p>
 
-<h2>KeymanWeb</h2>
+<h2>Implementation Note</h2>
 
-<p>Of particular note is KeymanWeb, our web-based keyboarding solution designed to work in browsers on any platform or device, touch-enabled or not.
-  As KeymanWeb is also embedded within the iOS and Android apps as their keyboarding engine due to its flexibility, ensuring full compatibility
-  with KeymanWeb will go a long way to making your keyboards fully cross-platform compatible.  Most standard Keyman
-  functionality is available, though there are certain additional restrictions due to historical web development
-  conditions.</p>
+<p>Internally, Keyman for iPhone and iPad and Keyman for Android use KeymanWeb to display the touch keyboard, and to process
+  rules. KeymanWeb keyboards (.js format) support basically all the same functionality as Keyman Desktop keyboards (.kmx format).
+  There are a few statements specific to KeymanWeb that are detailed below.</p>
```

Comment:
For clarity in tying this to the "Additional Stores for KeymanWeb" section, maybe either:
```suggestion
  There are a few store() statements specific to KeymanWeb that are detailed below.</p>
```

or move the statement to lead into the next.